source/_integrations/zwave_js.markdown

```diff
@@ -47,6 +47,18 @@ Note: A new network key is automatically generated for you. If this Z-Wave stick
 
 If you do not run Home Assistant OS (the default installation type) or Home Assistant Supervised, please see the [advanced installation instructions](#advanced-installation-instructions).
 
+### Discovery via USB
+
+Some devices can be auto-discovered, which can simplify the Z-Wave JS setup process. The following devices have been tested with discovery, and offer a quick setup experience:
+
+| Device | Identifier | Vendor |
+| -------| ---------- | ------ |
+| Aeotec Z-Stick Gen5+ | 0658:0200 | https://aeotec.com/z-wave-usb-stick/ |
+| Nortek HUSBZB-1 | 10C4:8A2A | https://www.nortekcontrol.com/products/2gig/husbzb-1-gocontrol-quickstick-combo/ |
+| Zooz ZST10 | 10C4:EA60 | https://www.getzooz.com/zooz-zst10-s2-stick.html |
```

Comment:
We should not use bare URLs, they render really badly. Instead, link the device names maybe?